.github/workflows/rust.yml

```diff
@@ -2,9 +2,7 @@ name: Rust
 
 on:
   push:
-    branches: [ "main" ]
-  pull_request:
-    branches: [ "main" ]
+    branches: "**"
 
```

Comment:
Why not on PRs?

Reply:
Because it would run twice then one for the pull request and one for the branch